packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -893,6 +893,12 @@ class _CupertinoNavigationBarState extends State<CupertinoNavigationBar> {
 /// ** See code in examples/api/lib/cupertino/nav_bar/cupertino_sliver_nav_bar.0.dart **
 /// {@end-tool}
 ///
+/// {@tool dartpad}
+/// To add a widget to the bottom of the nav bar, wrap it with [PreferredSize] and provide its fully extended size
```

Comment:
Nit: Documentation needs to have all sentences with proper punctuation.
```suggestion
/// To add a widget to the bottom of the nav bar, wrap it with [PreferredSize] and provide its fully extended size.
```